go/engine/passphrase_recover.go

```diff
@@ -218,7 +218,8 @@ func (e *PassphraseRecover) loginWithPaperKey(mctx libkb.MetaContext) (err error
 
 	if err := e.changePassword(mctx); err != nil {
 		// Log out before returning
-		if err2 := RunEngine2(mctx, NewLogout()); err2 != nil {
+		// TODO FIX ME
+		if err2 := RunEngine2(mctx, NewLogout(libkb.LogoutOptions{})); err2 != nil {
```

Comment:
not sure...why are we logging out on errors here? seems bad

Reply:
I think this is only in the code path where you are initially logged out, then you log in using paper key, and this lets you change your passphrase. If change fails, it wants to log you out so you can get to the initial state.

Reply:
So what options do we want here? Kill secrets (since you have a passphrase) and force?

Reply:
I think this is good actually, because `NewLoginWithPaperKey` logs you in and stores secrets in keychain (maybe it shouldn't? i couldn't figure out exactly), so we want to clear after ourselves here if anything fails

Reply:
So yeah killSecrets and force